.drone.star

```diff
@@ -0,0 +1,74 @@
+OC_CI_NODEJS = "owncloudci/nodejs:18"
+OC_CI_BUILDIFIER = "owncloudci/bazel-buildifier:latest"
+
+def main(ctx):
+    return checkStarlark() + unitTestPipeline()
+
+def checkStarlark():
+    return [{
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "check-starlark",
+        "steps": [
+            {
+                "name": "format-check-starlark",
+                "image": OC_CI_BUILDIFIER,
+                "commands": [
+                    "buildifier --mode=check .drone.star",
+                ],
+            },
+            {
+                "name": "show-diff",
+                "image": OC_CI_BUILDIFIER,
+                "commands": [
+                    "buildifier --mode=fix .drone.star",
+                    "git diff",
+                ],
+                "when": {
+                    "status": [
+                        "failure",
+                    ],
+                },
+            },
+        ],
+        "depends_on": [],
+        "trigger": {
+            "ref": [
+                "refs/pull/**",
+            ],
+        },
+    }]
+
+def unitTestPipeline():
+    return [{
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "unit-tests",
+        "steps": installPnpm() +
+                 [
+                     {
+                         "name": "unit-tests",
+                         "image": OC_CI_NODEJS,
+                         "commands": [
+                             "pnpm run coverage",
```

Comment:
The pipe line says running unit tests but why there is only command for running coverage only? or the it runs unit tests?

Reply:
`pnpm run coverage` runs the unit and coverage